### DISCUSSION TITLE
Is it possible to use ruskdesk on just lan?

### DISCUSSION THREAD

**Discussion by Darin755** [rustdesk/rustdesk#5584] [Q&A]
I'm looking for a alternative to vnc and I want to setup rustdesk to only except a direct connection. Is this possible?

**Comment by SergeyMy** [comment]
just create your servers in local network and enable direct access by IP
PS
0. with audit and address book (if you also have your own and API server)
IP access will be instant, but without encryption.
access by ID will be delayed but with the ability to:
1. Encryption
2. relaying via relay

**Comment by necros2k7** [comment]
Anyone compared with VNC, is it worth it in LAN only?
